packages/functional-tests/tests/misc/authClientV2.spec.ts

```diff
@@ -44,20 +30,13 @@ test.describe('auth-client-tests', () => {
 
   test.beforeEach(({}, { project }) => {
     test.skip(project.name === 'production');
-
-    email = `signin${crypto.randomBytes(8).toString('hex')}@restmail.net`;
-    password = `${crypto.randomBytes(10).toString('hex')}`;
```

Comment:
The email and password use `crypto.randomBytes`, but the email fixture uses `Math.random()`. Do we need to use `crypto.randomBytes`?

Reply:
crypto.randomBytes is provides 'cryptographically secure randomness', ie better random than math.random, so IMO, it's slightly better.